Client/Application/AppDelegate.swift

```diff
@@ -157,12 +157,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        LeanplumIntegration.sharedInstance.setup(profile: profile)
+
         // We need to check if the app is a clean install to use for
         // preventing the What's New URL from appearing.
         if getProfile(application).prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
             getProfile(application).prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
+            LeanplumIntegration.sharedInstance.track(event: "First Run")
```

Comment:
Stefan, since the events are consumed by Product/Marketing, in the past I've recommended human readable names. The important thing is that we are consistent and we advertise a 'legend' to Product/Marketing (so that is why an enum would help, as @farhanpatel suggested), so if you feel strongly about no spaces, please let us know.